### DISCUSSION TITLE
함수형 컴포넌트 코드

### DISCUSSION THREAD

**Discussion by ghksrl6582** [fsoftwareengineer/todo-application#40] [질문과 답변] [answered]
안녕하세요 작가님 책 잘보고 있습니다.

제가 못찾는건지 함수형 코드를 못찾겠습니다...

**Comment by fsoftwareengineer** [comment, marked as answer]
~~안녕하세요, 함수형 코드는 곧 업로드 될 예정입니다. 업로드 되는대로 코멘트 남기도록 하겠습니다.~~

함수형 코드는 https://github.com/fsoftwareengineer/todo-application/tree/functional-components 에서 확인 해 주시기 바랍니다.

**Reply by ghksrl6582** [reply]
감사합니다!
